scripts/tasks/task_compileenv.jl

```diff
@@ -1,4 +1,59 @@
 import Pkg, Libdl, PackageCompiler
+import TOML
+
+
+const config_fname = "vscode-compileenv.toml"
+
+"""
+    find_dev_packages(envdir::AbstractString)
+
+Locate the packages that are in `dev` mode for a given project environment.
+"""
+function find_dev_packages(envdir::AbstractString)
+    fname = joinpath(envdir, "Manifest.toml")
+    !isfile(fname) && return Symbol[]
+    devpkgs = Symbol[]
+    parsed = TOML.parse(read(fname, String))
+    for key in keys(parsed)
+        sub = parsed[key]
+        "path" in keys(sub[1]) && push!(devpkgs, Symbol(key))
+    end
+    devpkgs
+end
+
+
+"""
+    read_configuration(envdir)
+
+Read the configuration file `vscode-compileenv.toml` from the environment directory.
+
+This file should look like:
+
+```
+[sysimage]
+excluded_packages=[]   # Additional packages to be exlucded in the system image
+precompile_statements_file=[]  # Precompile statements file to be used
+precompile_execution_file=[] # Precompile execution file to be used
```

Comment:
```suggestion
exclude = []   # Additional packages to be exlucded in the system image
statements_files = []  # Precompile statements file to be used
execution_files = [] # Precompile execution file to be used
```
maybe?

Reply:
good idea! I have applied the changes in the code.